src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/RegexRunner.cs

```diff
@@ -184,7 +185,7 @@ protected internal virtual void Scan(ReadOnlySpan<char> text)
                 }
 
                 runmatch = null;
-                match.Tidy(runtextpos, 0);
```

Comment:
line 182 -- `if (mode == RegexRunnerMode.Existence)` ?

Reply:
I don't understand the comment.  Can you elaborate?